app/experimenter/experiments/views.py

```diff
@@ -233,3 +233,15 @@ def form_invalid(self, form):
 
 class ExperimentRapidView(TemplateView):
     template_name = "experiments/rapid.html"
+
+
+class ExperimentNimbusUIView(TemplateView):
+    template_name = "experiments/nimbus-ui.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context["nimbus_dev"] = settings.NIMBUS_DEV
+        context["app_config"] = json.dumps({
+            "sentry_dsn": settings.SENTRY_DSN,
+        })
+        return context
```

Comment:
I already did this for you 😃 

https://github.com/mozilla/experimenter/blob/main/app/experimenter/base/context_processors.py#L14

Reply:
@jaredlockhart can you elaborate on how this is used? My understanding is this makes each key available in the template (e.g. `SENTRY_DSN` or `FEATURE_MESSAGE_TYPE`), but I'm looking for the whole object to be readily available at render time so I can encode the whole thing. Maybe I'm misunderstanding.

Reply:
Unless we want each unique setting to be its own data-attribute? I can get behind that

Reply:
Yeah exactly, this exposes it as a template var, so you could pass it in as a data attribute, or if you wanna pass it in as a whole object you could either do what you were doing with dumping it into JS here the same way you were doing in the view, or you can build the object in the template like

```js
<script>
  const config = {
    SENTRY_DSN: "{{ SENTRY_DSN }}",
  };
  App.init(config);
</script>
```

Or if you wanna only do it in the view you can just keep it how you have it in the view and we can rip it out of here, there's just no need to have it in one place.

Reply:
Okay cool. I think I'll avoid setting up an object and initializing from the page because we already init from the script automatically, and it would involve exposing an object that the page could call init on. But I did update to move the whole config under the context processor.